docker/docker-compose.yml

```diff
@@ -56,6 +54,4 @@ services:
       dockerfile: docker/viame_girder_worker.Dockerfile
 
 volumes:
-  mongo_db:
-  girder_assetstore:
```

Comment:
they can be explored with a file browser or rm'd.  Otherwise they go somewhere in `/tmp` that is both hard to find and difficult to confidently wipe out.

Reply:
This probably depends on OS, but I know docker data is usually stored in `/var/lib/docker`, and volumes are accessible through `docker volume ls` and `docker inspect <volume_name>`, which give you the list of volumes and the details (including file path) of that volume, respectively. Personally I think this is better left as docker volumes, as I've never needed to inspect the volumes themselves, and I feel it clutters the codebase. Plus you could always create another docker-compose override for your specific local environment.

Reply:
"let docker manage these" is I think a reasonable attitude, and I've been too lazy to figure out how to manage them through docker, so you forced me to learn something.  I'll revert these changes.